Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official Node.js 10 image.
+# https://hub.docker.com/_/node
+FROM node:18.12.1
+
+# Create and change to the app directory.
+WORKDIR /usr/src/app
+
+# Copy application dependency manifests to the container image.
+# A wildcard is used to ensure both package.json AND package-lock.json are copied.
+# Copying this separately prevents re-running npm install on every code change.
+COPY package*.json ./
+
+# Install production dependencies.
+RUN npm i --omit=dev
```

Comment:
How about having a deployment option for deploying a dev env, with swagger and everything?

Reply:
we agreed that we can run the server locally while working with postgres on gcp.